capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -1103,9 +1124,9 @@ public void onError(final ImportedFileValidationException exception) {
                         hideActivityIndicatorAndEnableInteraction();
                         final FileImportValidator.Error error = exception.getValidationError();
                         if (error != null) {
-                            showInvalidFileError(error);
-                        } else {
-                            showGenericInvalidFileError();
+                            Error errorClass = new Error(FileImportValidator.Error.valueOf(error.name()));
```

Comment:
[Request] Change in the requirements: please show a dialog using the `ErrorType`'s title string. Analog to the `showMultiPageLimitError()` method.